tests/TicketingSystem.PaymentIntegrationTests/PaymentProcessTests.cs

```diff
@@ -0,0 +1,231 @@
+﻿using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using TicketingSystem.BusinessLogic.Dtos;
+using TicketingSystem.BusinessLogic.Models;
+using TicketingSystem.Common.Enums;
+using TicketingSystem.IntegrationTests;
+using TicketingSystem.WebApi.Controllers;
+using TicketingSystem.WebApi.Models;
+using Xunit;
+
+namespace TicketingSystem.PaymentIntegrationTests
+{
+
+    public class PaymentProcessTests(DatabaseFixture fixture) : FixtureTestsBase(fixture), IClassFixture<DatabaseFixture>
+    {
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task GivenPaymentProcess_WhenPaymentSucceeded_ShouldUpdateSeatStateToSoldAndUpdatePaymentAndDeleteCartItems(bool paymentShouldBeSuccessful)
```

Comment:
This file seems to be a duplicate

Reply:
The PaymentIntegrationTests project was removed from the solution but not from the disk. Fixed